.github/workflows/version-bump.yml

```
@@ -167,13 +166,21 @@ jobs:
         if: ${{ steps.version-changed.outputs.changes_to_commit == 'TRUE' }}
         env:
           PR_BRANCH: ${{ steps.create-branch.outputs.name }}
-        run: git push -u origin $PR_BRANCH
+        run: git push -u origin $
```

```suggestion
        run: git push -u origin $PR_BRANCH
```

This looks like this is going to fail.  Is this still supposed to be `$PR_BRANCH`?